web/templates/base.html

```diff
@@ -63,7 +63,8 @@
 		<div id="push"></div>
 		</div>
 
-		<footer>
+		{% include 'layout/footer.html' %}
+<!--/.		<footer>
```

Comment:
so comments are no good practice? I love the CSS file, where I know where to look due to comments.:)